x-pack/test/security_solution_endpoint_api_int/apis/metadata.ts

```diff
@@ -58,17 +63,22 @@ export default function ({ getService }: FtrProviderContext) {
           const policyId = policy.integrationPolicies[0].policy_id;
           const currentTime = new Date().getTime();
 
+          const agentDocs = generateAgentDocs(currentTime, policyId);
+
           await Promise.all([
-            bulkIndex(getService, AGENTS_INDEX, generateAgentDocs(currentTime, policyId)),
+            bulkIndex(getService, AGENTS_INDEX, agentDocs),
             bulkIndex(getService, METADATA_DATASTREAM, generateMetadataDocs(currentTime)),
```

Comment:
ideally we'd use the data generator but since we're testing field filtering in these tests and the generator doesn't support override fields yet (some fields on the generated are read only too), it's not a straight forward replace. will consider for a separate PR.